ocs_ci/ocs/bucket_utils.py

```diff
@@ -2023,3 +2034,15 @@ def sample_if_objects_expired(mcg_obj, bucket_name, prefix="", timeout=600, slee
 
     assert sampler.wait_for_func_status(result=True), f"{message} are not expired"
     logger.info(f"{message} are expired")
+
+
+def s3_put_bucket_lifecycle_config(mcg_obj, buckets, rule):
+    """
+    This method applies lifecycle configuration for a bucket
+
+    """
```

Comment:
To improve accuracy and clarity:
```suggestion
def bulk_s3_put_bucket_lifecycle_config(mcg_obj, buckets, lifecycle_config):
    """
    This method applies a lifecycle configuration to multiple buckets
    
     Args:
     - mcg_obj: An MCG object containing the MCG S3 connection credentials
     - buckets (list): list of bucket names to apply the lifecycle rule to
     - lifecycle_config (dict): a dict following the expected AWS json structure of a config file
    """
```